example/server.js

```diff
@@ -32,11 +32,17 @@ function init (port, pgConfig, callback) {
           return reply(listTags);
         });
       }
+    }, {
+      method: 'GET',
+      path: '/getAllActive',
+      handler: function (request, reply) {
+        request.getAllActive(function (error, allTags) { //eslint-disable-line
```

Comment:
I think we should have a more descriptive name for this function as it will be used on the application with other tables.